tox.ini

```diff
@@ -13,6 +13,15 @@ commands =
     coverage erase
     py.test {posargs}
 
+[testenv:docs]
+basepython = python3.6
+setenv =
+    PYTHONHASHSEED = 0
+deps = -rdocs-requirements.txt
+commands =
+    sphinx-build -W -b html -d {envtmpdir}/doctrees docs docs/_build/html
+    sphinx-build -W -b coverage -d {envtmpdir}/doctrees docs docs/_build/html
```

Comment:
[Asserts doc coverage](http://www.sphinx-doc.org/en/stable/ext/coverage.html), but, not sure if it actually works.... I removed some docstrings just to try it out, and it still passed, but maybe I don't understand it fully.